pymess/models/emails.py

```diff
@@ -144,8 +144,8 @@ def clean_subject(self, context_data=None):
     def get_subject(self):
         return self.subject
 
-    def render_subject(self, context_data):
-        context_data = self._update_context_data(context_data)
+    def render_subject(self, context_data, recipient=None):
```

Comment:
also here...

Reply:
also here recipient in **clean_subject** does not exist, so it is optional